test/unit/spec/appWidgets/input/selectInputSpec.js

```diff
@@ -228,38 +228,38 @@ define(['common/runtime', 'widgets/appWidgets2/input/selectInput', 'testUtil'],
 
             const carrotItem = inputElem.querySelector('option[value="carrot"]');
             await TestUtil.waitForElementChange(carrotItem, () => {
-                bus.emit('disable-values', {
+                bus.emit('set-disabled-values', {
                     values: ['carrot'],
                 });
             });
             itemsDisabled.carrot = true;
             checkItems(itemsDisabled, inputElem);
         });
 
-        it('Should obey a message to enable selection options', async () => {
-            const config = Object.assign({}, testConfig, { disabledValues: ['carrot'] });
-            const widget = SelectInput.make(config);
+        // it('Should obey a message to enable selection options', async () => {
```

Comment:
are these tests going to be reinstated at some point? rewritten? what's the plan?

Reply:
Oops. I'm not really finished with the testing bits. Probably should've left as draft. I'll add more to the description. 

The main thrust here was that I felt that using `disable` vs. `enable` to switch on / off a few might be (1) confusing during use, and (2) a good way to get caught up in messaging mishaps, so I altered it a bit. Now, there's no `enable` call, and I changed `disable-values` to `set-disabled-values`, which changes the disabled set all together. 

Mainly, now, when there's an alteration to any of the selections, the state of what should be disabled just gets passed in bulk to all select inputs, instead of individually trying to figure out what's enabled or disabled.

Reply:
So, long winded way of saying this particular test is going away.